apps/ui/src/components/pages/application/collapse/CommentEntry.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2025 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { Flex, Typography } from 'antd';
+
+const { Text, Paragraph } = Typography;
+
+type CommentEntryProps = {
+	username: string;
+	comments: string;
+	submittedAt: Date | string;
+};
+
+const CommentEntry = ({ username, comments, submittedAt }: CommentEntryProps) => {
+	return (
+		<Flex gap={'middle'}>
+			<Flex vertical>
+				<Flex vertical gap={'middle'}>
+					<Flex align="center">
+						<Text style={{ fontSize: '15px' }} strong>
+							{`${username}, ${formatToShortDate(submittedAt)}`}
+						</Text>
+					</Flex>
+				</Flex>
+				<Flex>
+					<Paragraph style={{ fontSize: '13px' }}>{comments}</Paragraph>
+				</Flex>
+			</Flex>
+		</Flex>
+	);
+};
+
+export default CommentEntry;
+
+/**
+ * Formats a Date or date string into MM/dd/yy format
+ * @param date - Date object or date string to format
+ * @returns Formatted date string in MM/dd/yy format
+ */
+export function formatToShortDate(date: Date | string): string {
+	const dateObj = typeof date === 'string' ? new Date(date) : date;
+
+	// Check if the date is valid
+	if (isNaN(dateObj.getTime())) {
+		return '';
+	}
+
+	const month = String(dateObj.getMonth() + 1).padStart(2, '0'); // getMonth() is 0-indexed
+	const day = String(dateObj.getDate()).padStart(2, '0');
+	const year = String(dateObj.getFullYear()).slice(-2); // Get last 2 digits of year
+
+	return `${month}/${day}/${year}`;
+}
```

Comment:
I'd move this definition before CommentEntry as well since it's our default export